share/namespace.go

```diff
@@ -130,10 +130,7 @@ func (n Namespace) ValidateForBlob() error {
 	if err := n.ValidateForData(); err != nil {
 		return err
 	}
-	if bytes.Compare(n, MaxPrimaryReservedNamespace) < 1 {
-		return fmt.Errorf("invalid blob namespace(%s): reserved namespaces are forbidden", n)
-	}
-	if bytes.Compare(n, MinSecondaryReservedNamespace) > -1 {
+	if n.ToAppNamespace().IsReserved() {
```

Comment:
Let's avoid conversions and copy code for now.

Reply:
It seems like a burden to maintain the same validation logic in two separate repos but defer to maintainers.

Resolved via f0f02f1a47e67f0c669291a775730fdf8750ef05